### ISSUE TITLE
Interesting python scripts

### ISSUE THREAD

**Issue by prajjyadav** [prajjyadav/Python-Scripts#3]
contribute some interesting python scripts.
You can even create your own issue with the name of the scripts or piece of code you want to write and do add comments as to what this script would do in the starting of file.
Avoid making PR for hello-world.
No need to make changes in README.md

**Comment by juanbelieni** [comment]
How about a tic tac toe game?

**Comment by prajjyadav** [comment]
Yeah, why not.... Create a new issue and start working on it... try level up and make it unbeatable in AI mode. Happy Hacking :)

**Comment by ghost** [comment]
Could I add a script that analyzes a text file and spits out unique words (removing punctuation and lowercasing every word to only get unique words, not word variants)?

I just saw this has been done in #36 already, I might come up with something different

**Comment by prajjyadav** [comment]
yeah sure.. create an issue and go ahead :)

**Comment by tdchua** [comment]
Hi! I would like to write a script that moves your mouse and clicks somewhere on your screen to prevent the computer from going idle!

**Comment by skate1512** [comment]
Hi! I would like to submit a script that saves a random wiki article to a txt file.

**Comment by zeeshanok** [comment]
Hello! I would like to make a random number guesser game in python. May I create the issue for it?
